### ISSUE TITLE
Error in brew...: No valid input

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/infographiq#2]
I'm tring to build info_fk so this could be an issue over there or this might be my unfamiliarity with installing R packages, but I'm trying to make sense of this error when trying to run `create_info_site()`:

```
> create_info_site()
Loading tidyverse: ggplot2
Loading tidyverse: tibble
Loading tidyverse: tidyr
Loading tidyverse: readr
Loading tidyverse: purrr
Loading tidyverse: dplyr
Conflicts with tidy packages ---------------------------------------------------
filter(): dplyr, stats
lag():    dplyr, stats
Parsed with column specification:
cols(
  svg = col_character(),
  svg_id = col_character(),
  label = col_character(),
  status_text = col_character(),
  status_color = col_character()
)
Error in brew(scene_brew, file.path(path_rmd, rmds[i])) : No valid input
```

This is after loading infographiq from my local repo using devtools (`> load_all('../infographiq')`).

Also uhhh... I think this is the traceback:
```
> traceback()
3: stop("No valid input")
2: brew(scene_brew, file.path(path_rmd, rmds[i])) at create_info_site.R#133
1: create_info_site()
```

**Comment by 7yl4r** [comment]
Ah. Here's the issue. `scene_brew == ""` because you can't `system.file('site_template/scene.rmd.brew', package='infographiq')` if the package is just loaded and not installed.

**Comment by 7yl4r** [comment]
The solution would be install it rather than use `load_all`, but I can't seem to get that working.

Hey @mhepner90 : Have you installed an R package from a local source before? 

I have read through several tutorials using `devtools::install()` and `devtools::install_local()` but somehow `library('infographiq')` is not loading the right code. I am not even sure how that is possible.

**Comment by mhepner90** [comment]
So there is a trail of our solution this morning... You need to have the latest version of R as well as have library servr for the package to load.

**Comment by 7yl4r** [comment]
closed by 49239aa0a242d109500e178a3f487fecc65e5df5
